packages/algoliasearch/src/builds/node.ts

```diff
@@ -175,6 +175,142 @@ import { createUserAgent, RequestOptions } from '@algolia/transporter';
 
 import { AlgoliaSearchOptions, InitAnalyticsOptions, InitRecommendationOptions } from '../types';
 
+export {
```

Comment:
Note: Types are not being exported in `browser.ts`.

Reply:
Yes, from what I saw there is no declaration file created from  `browser.ts`.
The entrypoints for the API Extractor are only `node.d.ts` (`api-extractor.json`) and `browserLite.d.t` (`api-extractor-lite.json`)

Reply:
I see, let it this way then.